identity/app/views/fragments/form/switch.scala.html

```diff
@@ -10,7 +10,7 @@
     footer: Option[Html] = None
 )(implicit handler: views.html.helper.FieldConstructor, messages: play.api.i18n.Messages)
 
-<label class="
+<label role="checkbox" class="
```

Comment:
Could you elaborate a bit, which this is a good idea?

From MDN

> The checkbox role is used for checkable interactive controls. If an element uses role="checkbox" it is required for that element to also have a aria-checked attribute that exposes the checkbox's state to assistive technology.

Reply:
it's not, i brainfarted a bit there